FloconDesktop/domain/src/commonMain/kotlin/io/github/openflocon/domain/network/models/FloconNetworkCallDomainModel.kt

```diff
@@ -73,9 +81,15 @@ data class FloconNetworkRequestDomainModel(
 )
 
 data class FloconNetworkResponseDomainModel(
+    val floconCallId: String? = null,
```

Comment:
as it's a domain model

you should not have a nullable `floconCallId` and `floconNetworkType` 

if the data model does not contains this, returns null in the mapper